### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by ghost** [gokadzev/Musify#237] [Q&A] [answered]
Can anyone provide me with a windows binary for this app, I've tried compiling it locally and on github spaces but i've been having trouble

**Comment by gokadzev** [comment]
> Can anyone provide me with a windows binary for this app, I've tried compiling it locally and on github spaces but i've been having trouble

Actually, I updated the PC branch a long time ago, specifically for Linux and Windows (although I haven't tested it for Windows), tell me which branch are you compiling and what problem you have and i will try to help you

**Reply by ghost** [reply]
I am compiling on the PC branch
On compile I ran:

`flutter upgrade`
`flutter clean`
`flutter pub get`
// this resulted in the error:
```Resolving dependencies...
Because musify depends on flutter_localizations from sdk which depends on intl 0.18.0, intl 0.18.0 is required.
So, because musify depends on intl ^0.17.0, version solving failed.
```
// I then edited the pubsec.yaml file to make the "^0.17.0" to "0.18.0"
// I ran it again
`flutter pub get`
// no errors
`flutter build windows`
// I got this error:
```
windows/runner/CMakeLists.txt does not define version information, updating.
windows/runner/Runner.rc does not define use Flutter version information, updating.
../../AppData/Local/Pub/Cache/hosted/pub.dev/package_info_plus-3.0.2/lib/src/file_version_info.dart(13,17): error G99FC7582: Field 'wLanguage' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`. [C:\Users\Solarized-Incog\Downloads\Musify\build\windows\flutter\flutter_assemble.vcxproj]
../../AppData/Local/Pub/Cache/hosted/pub.dev/package_info_plus-3.0.2/lib/src/file_version_info.dart(16,17): error G99FC7582: Field 'wCodePage' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`. [C:\Users\Solarized-Incog\Downloads\Musify\build\windows\flutter\flutter_assemble.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(247,5): error MSB8066: Custom build for 'C:\Users\Solarized-Incog\Downloads\Musify\build\windows\CMakeFiles\320914534f5b8f16905c4759ecdb6a26\flutter_windows.dll.rule;C:\Users\Solarized-Incog\Downloads\Musify\build\windows\CMakeFiles\7f431143e3a3cc7201d18b2e4ca51d05\flutter_assemble.rule;C:\Users\Solarized-Incog\Downloads\Musify\windows\flutter\CMakeLists.txt' exited with code 1. [C:\Users\Solarized-Incog\Downloads\Musify\build\windows\flutter\flutter_assemble.vcxproj]
Building Windows application...                                    32.5s
Build process failed.
``` 
I tried downloading the .exe from the github action that you ran for windows, but the file expired so github doesn't store it anymore
I also tried running the github action using a fork with different workflows including the original but I keep getting errors on the actions as well

**Reply by gokadzev** [reply]
remove package_info_plus (and its uses in code) from pubspec.yaml and run: flutter pub get. then try again to build

**Reply by gokadzev** [reply, marked as answer]
@BlueIncog I've made some changes so you can try now (If i have free time, I will upstream changes from V5.3.0 too :) )

**Reply by ghost** [reply]
thank you very much, I really appreciate it 🙏
